src/Command/PqCommands/ContentHubPqNonTranslatables.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace Acquia\Console\ContentHub\Command\PqCommands;
+
+use Acquia\Console\ContentHub\Command\Helpers\DrupalServiceFactory;
+use Drupal\Core\Config\Entity\ConfigEntityInterface;
+use Drupal\Core\Entity\EntityTypeBundleInfoInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * Runs check against non-translatable entities.
+ */
+class ContentHubPqNonTranslatables extends ContentHubPqCommandBase {
+
+  /**
+   * The Drupal service factory service.
+   *
+   * @var \Acquia\Console\ContentHub\Command\Helpers\DrupalServiceFactory
+   */
+  protected $serviceFactory;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $defaultName = 'ach:pq:non-translatables';
+
+  /**
+   * Constructs a new ContentHubPqNonTranslatables object.
+   *
+   * @param \Acquia\Console\ContentHub\Command\Helpers\DrupalServiceFactory $drupalServiceFactory
+   *   The Drupal service factory service.
+   * @param string|null $name
+   *   The name of this command.
+   *
+   * @throws \Exception
+   */
+  public function __construct(DrupalServiceFactory $drupalServiceFactory, string $name = NULL) {
+    parent::__construct($name);
+    $this->serviceFactory = $drupalServiceFactory;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function configure() {
+    parent::configure();
+    $this->setDescription('Analyzes non-translatable entities and bundles.');
+    $this->setAliases(['ach-pq-nt']);
+    $this->addOption('entity-type', 'e', InputOption::VALUE_OPTIONAL, 'Run checks for the provided entity type. Example node,user,paragraph', 'node');
+    $this->addUsage('ach:pq:non-translatables --entity-type "node,paragraph"');
+  }
+
+  /**
+   * {@inheritDoc}
+   *
+   * @throws \Exception
+   */
+  protected function runCommand(InputInterface $input, PqCommandResult $result): int {
```

Comment:
We'd only want to go through content entities.

Reply:
fixed in [6f47d59](https://github.com/acquia/contenthub-console/pull/35/commits/6f47d59a503068b9c275aabd01d10f35820bec72)